src/main/java/com/volmit/iris/engine/object/IrisObject.java

```diff
@@ -892,20 +884,6 @@ public void rotate(IrisObjectRotation r, int spinx, int spiny, int spinz) {
         states = dx;
     }
 
-    public void place(Location at) {
-        for (BlockVector i : getBlocks().keySet()) {
-            Block b = at.clone().add(0, getCenter().getY(), 0).add(i).getBlock();
-            b.setBlockData(Objects.requireNonNull(getBlocks().get(i)), false);
-
-            if (getStates().containsKey(i)) {
-                Iris.info(Objects.requireNonNull(states.get(i)).toString());
-                BlockState st = b.getState();
```

Comment:
This is fine, if my assumption is that we are always using an IObjectPlacer which just places in the world. then this is fine to remove